build-tools/xaprepare/xaprepare/Steps/Step_DownloadMonoArchive.cs

```diff
@@ -36,19 +36,6 @@ async Task<bool> DownloadMonoArchive (Context context)
 				return false;
 			}
 
-			Log.StatusLine ("Checking if all runtime files are present");
-			var allRuntimes = new Runtimes ();
-			if (MonoRuntimesHelpers.AllBundleItemsPresent (allRuntimes)) {
```

Comment:
We might actually want to keep this check, just rename it. It would check if all the **archive** files are present to decide whether we need to download or reinstall it.

Reply:
I've updated this check and also added it to the new InstallMonoRuntimes step.